pymeasure/instruments/attocube/anc300.py

```diff
@@ -290,9 +272,9 @@ def __init__(
         # switch console echo off
         self.ask('echo off')
 
-    def check_errors(self):
+    def check_set_errors(self, context=""):
```

Comment:
This method should return a list of errors, not nothing. Same I think for others below.

Reply:
See the lower part of https://github.com/pymeasure/pymeasure/issues/667#issuecomment-1546647620, too -- e.g. this should probably do a `read()` without context, catch an occurring ValueError, log that (including the `context` known in here, and turn this into an error list to return. Or re-raise after logging, as you prefer.

Reply:
Done